tests/convert-tests/026-rollback-output/test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Make sure "btrfs-convert -r" is outputting the correct filename
+
+source "$TEST_TOP/common" || exit
+source "$TEST_TOP/common.convert" || exit
+
+setup_root_helper
+prepare_test_dev
+
+check_global_prereq mkfs.ext4
+check_prereq btrfs-convert
+check_prereq btrfs
+
+convert_test_prep_fs ext4 mke2fs -t ext4 -b 4096
+run_check_umount_test_dev
+convert_test_do_convert
+
+tmp=$(_mktemp convert-rollback)
+# Rollback and save the output.
+run_check_stdout "$TOP/btrfs-convert" --rollback "$TEST_DEV" >> "$tmp"
+
+if ! grep -q "ext2_saved/image" "$tmp"; then
```

Comment:
Using a temporary file is ok, it makes the code a bit more readable, otherwise it can be also in a pipe like is eg. in tests/cli-tests/002-balance-full-no-filters/test.sh:
```
 20 run_check_stdout $SUDO_HELPER "$TOP/btrfs" balance start --background "$TEST_MNT" |
 21         grep -F "Full balance without filters requested." >/dev/null ||
 22         _fail "full balance warning not in the output"
```

Reply:
Normally I'd prefer avoid long pipes (IIRC some bash setting can change the return value for long pipes), but in this particular case, it's only two commands and I'm fine to put them into the `if` condition.

Will fix it in the next (and hopefully the last) update.